acquia/acquia_cms
# ACMS-1140: Updated the tour module as per the latest discussion.

**Motivation**
* The tour module isolation test shows /admin/tour/dashboard page as blank page if no helper modules are present.
* Also, the logo is being fetched from acquia_cms profile which is incorrect as the module should work in isolation too.
Fixes #NNN

**Proposed changes**
* Updated code to show a message to user in case no helper modules are present.
* Updated code to fetch the site logo instead of Acquia CMS logo for isolated environments.

**Alternatives considered**
* None

**Testing steps**
* Install Drupal site and create a custom profile.
* install the acquia_cms_tour module on the site and visit the page - /admin/tour/dashboard.
* The dashboard page should be accessible without any errors and should show a message for user to enable helper modules if not enabled.
* If the helper modules are enabled the message to enable will not be shown and the forms would be displayed normally.

For more details, check the recording here - https://www.loom.com/share/0f838724615c4c08a11378eb928030d8

**Merge requirements**
